LRCI-6176 Use property to enable/disable dist artifact sharing on AWS

kenjiheigel opened the pull request:
https://liferay.atlassian.net/browse/LRCI-6176

BryanEngler committed:
LPD-50745 portal-search-elasticsearch7-impl: Allow entryClassName as a sort field

https://liferay.atlassian.net/browse/LPD-50745

BryanEngler committed:
LPD-50745 portal-search-opensearch2-impl: Allow entryClassName as a sort field

https://liferay.atlassian.net/browse/LPD-50745

BryanEngler committed:
LPD-50745 portal-search-solr8-impl: Allow entryClassName as a sort field

https://liferay.atlassian.net/browse/LPD-50745

andrea-ale-sbarra committed:
LPD-51865 add custom fields support to CPConfigurationList entity

andrea-ale-sbarra committed:
LPD-51865 added requested filters to CPConfigurationList rest API

andrea-ale-sbarra committed:
LPD-51865 autogenerated code (gw buildREST)

rodrigoguedes committed:
LPD-50745 portal-search-solr8-impl: Add unit test for SearchSolrQueryAssemblerImpl

https://liferay.atlassian.net/browse/LPD-50745

rodrigoguedes committed:
LPD-50745 portal-search-elasticsearch7-impl: Add unit test for SortTranslator

https://liferay.atlassian.net/browse/LPD-50745

rodrigoguedes committed:
LPD-50745 portal-search-opensearch2-impl: Add unit test for SortTranslator

https://liferay.atlassian.net/browse/LPD-50745

CaioERP committed:
LPD-58598 add ERC to documentPreviewed event

CaioERP committed:
LPD-58598 add ERC in documentDownloaded event

CaioERP committed:
LPD-58598 test update to meet changes

interaminense committed:
LPD-56439 Improve readability for debugging the context

brianikim committed:
LPD-61185 Autogenerated Code (baseline)

brianikim committed:
LPD-61185 limit initial screenName autogenerated based on email to RFC 5321 limit

Mikellorza committed:
LPD-62098 Change view history toolbar title as we have in the designs

brianikim committed:
LPD-61185 Add new exception for screen name max length validation

brianikim committed:
LPD-61185 Enforce screen name max length validation

brianikim committed:
LPD-61185 Add new language key for screen name max length validation

brianikim committed:
LPD-61185 Apply usage of screen name max length validation

agnosmelo committed:
LRSD-1334 Add useCheckUploadAccess hook

brianikim committed:
LPD-61185 Integration Test

andrea-ale-sbarra committed:
LPD-51865 autogenerated code (gw buildService)

# breaking

## What modules/apps/commerce/commerce-product-service/src/main/java/com/liferay/commerce/product/service/impl/CPConfigurationListLocalServiceImpl.java

modules/apps/commerce/commerce-product-service/src/main/java/com/liferay/commerce/product/service/impl/CPConfigurationListLocalServiceImpl.java add addOrUpdate and update methods signature now expose ServiceContext

## Why

this change is required to support Custom Fields

----

# breaking

## What modules/apps/commerce/commerce-product-service/src/main/java/com/liferay/commerce/product/service/impl/CPConfigurationListServiceImpl.java

modules/apps/commerce/commerce-product-service/src/main/java/com/liferay/commerce/product/service/impl/CPConfigurationListServiceImpl.java add addOrUpdate and update methods signature now expose ServiceContext

## Why

this change is required to support Custom Fields

----

vendeltoreki committed:
LPD-61662 Even though a Layout is not published, its SiteNavigationMenuItem still can be included in the LAR

vendeltoreki committed:
LPD-61662 Export the referred Layout as disposable dependency. This way, it will not fail during the import.

magjed4289 committed:
LPD-61755 Filter active administrators

magjed4289 committed:
LPD-61755 Test

DavysonMelo committed:
LPD-53471 Add View All Steps locator

DavysonMelo committed:
LPD-53471 Add Performance by step page

DavysonMelo committed:
LPD-53471 Import Performance by step page into workflow pages

DavysonMelo committed:
LPD-53471 Move poshi to playwright

BarbaraCabrera committed:
LPD-61879 Update namespace in the editable values when duplicating fragments

interaminense committed:
LPD-60567 Check Analytics instance before use it

interaminense committed:
LPD-60567 Refactor CSS to be splitted by performance content

Co-authored-by: Rafaella Jordao <rafaella.jordao@liferay.com>
Co-authored-by: Tiago Campelo <tiago.campelo@liferay.com>

interaminense committed:
LPD-60567 Use the same pattern from CSM to fetch endpoints

Co-authored-by: Rafaella Jordao <rafaella.jordao@liferay.com>
Co-authored-by: Tiago Campelo <tiago.campelo@liferay.com>

interaminense committed:
LPD-60567 Organize components by cms and content dashboard folders

Co-authored-by: Rafaella Jordao <rafaella.jordao@liferay.com>
Co-authored-by: Tiago Campelo <tiago.campelo@liferay.com>

interaminense committed:
LPD-60567 Update unit tests

Co-authored-by: Rafaella Jordao <rafaella.jordao@liferay.com>
Co-authored-by: Tiago Campelo <tiago.campelo@liferay.com>

interaminense committed:
LPD-60567 Fix utils to be exported

Co-authored-by: Rafaella Jordao <rafaella.jordao@liferay.com>
Co-authored-by: Tiago Campelo <tiago.campelo@liferay.com>

interaminense committed:
LPD-60567 Adapt Context to the new structure

Co-authored-by: Rafaella Jordao <rafaella.jordao@liferay.com>
Co-authored-by: Tiago Campelo <tiago.campelo@liferay.com>

interaminense committed:
LPD-60567 Create CMS performance content

Co-authored-by: Rafaella Jordao <rafaella.jordao@liferay.com>
Co-authored-by: Tiago Campelo <tiago.campelo@liferay.com>

interaminense committed:
LPD-60567 Create Content Dashboard  performance content

Co-authored-by: Rafaella Jordao <rafaella.jordao@liferay.com>
Co-authored-by: Tiago Campelo <tiago.campelo@liferay.com>

interaminense committed:
LPD-60567 Export components

Co-authored-by: Rafaella Jordao <rafaella.jordao@liferay.com>
Co-authored-by: Tiago Campelo <tiago.campelo@liferay.com>

interaminense committed:
LPD-60567 Add missing packages

Co-authored-by: Rafaella Jordao <rafaella.jordao@liferay.com>
Co-authored-by: Tiago Campelo <tiago.campelo@liferay.com>

interaminense committed:
LPD-60567 Add new empty state svg

Co-authored-by: Rafaella Jordao <rafaella.jordao@liferay.com>
Co-authored-by: Tiago Campelo <tiago.campelo@liferay.com>

interaminense committed:
LPD-60567 Add new languages

Co-authored-by: Rafaella Jordao <rafaella.jordao@liferay.com>
Co-authored-by: Tiago Campelo <tiago.campelo@liferay.com>

interaminense committed:
LPD-60567 Import analytics reports js components web

Co-authored-by: Rafaella Jordao <rafaella.jordao@liferay.com>
Co-authored-by: Tiago Campelo <tiago.campelo@liferay.com>

interaminense committed:
LPD-60567 Reuse utils from analytics reports js components web

Co-authored-by: Rafaella Jordao <rafaella.jordao@liferay.com>
Co-authored-by: Tiago Campelo <tiago.campelo@liferay.com>

interaminense committed:
LPD-60567 Fix unit tests on site cms

Co-authored-by: Rafaella Jordao <rafaella.jordao@liferay.com>
Co-authored-by: Tiago Campelo <tiago.campelo@liferay.com>

interaminense committed:
LPD-60567 Rename performance content

Co-authored-by: Rafaella Jordao <rafaella.jordao@liferay.com>
Co-authored-by: Tiago Campelo <tiago.campelo@liferay.com>

interaminense committed:
LPD-60567 buildLang

bcarvalho89 committed:
LPD-61847 Improve members summary content styles

bcarvalho89 committed:
LPD-61847 Fix owner label for members mmodal in all spaces FDS

JavierMoral committed:
LPD-61906 Add scope to ERCInfoItemIdentifier

bcarvalho89 committed:
LPD-61847 Pass canManageMembers permission to SpaceSummaryHeader and ViewAllSpacesDisplayContext

CaioERP committed:
LPD-58598 add GetERC method

CaioERP committed:
LPD-58598 SF

CaioERP committed:
LPD-58598 update method to get the correct ERC

CaioERP committed:
LPD-58598 update dependencie

bcarvalho89 committed:
LPD-61847 Hide remove member button if canManageMembers is false

bcarvalho89 committed:
LPD-61847 Hide add/remove space members if canManageMembers is false

bcarvalho89 committed:
LPD-61847 Passing down the canManageMembers prop to members modal

CaioERP committed:
LPD-58598 sort

bcarvalho89 committed:
LPD-61847 Pass the canManageMembers prop to AddSpaceMembers

bcarvalho89 committed:
LPD-61847 Fix failing tests

bcarvalho89 committed:
LPD-61847 Auto SF

abfranca committed:
LRSD-10430 Fix swarm skills custom field got erased after linking a business event to a zendesk ticket

ruben-pulido committed:
LPD-59858 Add keywords and taxonomyCategoryItemExternalReferences to the list of field names to assert.

ruben-pulido committed:
LPD-59858 Set keywords and taxonomyCategoryItemExternalReferences when creating a random Site Page

ruben-pulido committed:
LPD-59858 Test patching keywords and taxonomyCategoryItemExternalReferences

ruben-pulido committed:
LPD-59858 Set keywords and taxonomyCategoryItemExternalReferences in SitePageDTOConverter

ruben-pulido committed:
LPD-59858 Set keywords and taxonomyCategoryItemExternalReferences when creating ServiceContext for adding and updating a Layout

jkappler committed:
LPD-61967 Move to init so that we can reuse

jkappler committed:
LPD-61967 Adds language key

jkappler committed:
LPD-61967 Change so that we pass the responseData to the refreshData

jkappler committed:
LPD-61967 Change message based on the responseData

jkappler committed:
LPD-61967 buildLang

andrea-ale-sbarra committed:
LPD-61185 autogenerated code (gw buildLang)

albertojml committed:
LPD-61120 BuildREST in test module

magjed4289 committed:
LPD-61755 Inline

magjed4289 committed:
LPD-61755 Change message

andrea-ale-sbarra committed:
LPD-51865 add custom fields support to CPConfigurationList entity (yaml file)

ruben-pulido committed:
LPD-59858 When calling _randomAssetCategories always generate at least one category, to avoid tests randomly passing despite categories not being correctly processed

ruben-pulido committed:
LPD-59858 Create vocabulary in global site, since its categories are being referenced both by pages created in the testGroup and in the irrelevantTestGroup.

Otherwise, when trying to create the site page in the irrelevant group an error is thrown, because the categories have been created in the testGroup, and since they are being referenced without scope, an error is thrown.

rodrigoguedes committed:
LPD-59404 portal-language-lang: Remove DevFF feature.flag.LPD-11232 from manual properties
https://liferay.atlassian.net/browse/LPD-59404

rodrigoguedes committed:
LPD-59404 portal-language-lang: Remove DevFF feature.flag.LPD-11232 from auto-generated properties
https://liferay.atlassian.net/browse/LPD-59404

rodrigoguedes committed:
LPD-59404 portal-impl: Remove DevFF feature.flag.LPD-11232 from portal.properties
https://liferay.atlassian.net/browse/LPD-59404

rodrigoguedes committed:
LPD-59404 portal-search-rest-impl: Replace DevFF LPD-11232 with Release FF LPS-179669
https://liferay.atlassian.net/browse/LPD-59404

rodrigoguedes committed:
LPD-59404 portal-search-rest-impl: Update openapi yaml replacing DevFF LPD-11232 with Release FF LPS-179669
https://liferay.atlassian.net/browse/LPD-59404

BarbaraCabrera committed:
LPD-61879 Add necessary integration test

rodrigoguedes committed:
LPD-59404 portal-search-rest-impl: Update openapi auto-generated files
https://liferay.atlassian.net/browse/LPD-59404

rodrigoguedes committed:
LPD-59404 portal-search-rest-test: Replace DevFF LPD-11232 with Release FF LPS-179669
https://liferay.atlassian.net/browse/LPD-59404

rodrigoguedes committed:
LPD-59404 analytics-cms-rest-test: Replace DevFF LPD-11232 with Release FF LPS-179669
https://liferay.atlassian.net/browse/LPD-59404

rodrigoguedes committed:
LPD-59404 playwright: Replace DevFF LPD-11232 with Release FF LPS-179669
https://liferay.atlassian.net/browse/LPD-59404

carolmariaabb committed:
LPD-62010 Use the CMS feature flag instead

carolmariaabb committed:
LPD-62010 Add new SUBSCRIBE permission for object entry folders

carolmariaabb committed:
LPD-62010 Add new service methods to subscribe and unsubscribe to object entry folders

carolmariaabb committed:
LPD-62010 Delete subscription when deleting the object entry folder

carolmariaabb committed:
LPD-62010 buildService

carolmariaabb committed:
LPD-62010 Add new endpoints to subscribe and unsubscribe to object entry folders

carolmariaabb committed:
LPD-62010 buildRest

carolmariaabb committed:
LPD-62010 Add new resource methods to subscribe and unsubscribe to object entry folders

carolmariaabb committed:
LPD-62010 Add new reusable methods following the same pattern as JournalFolderImpl and DLFolderImpl

carolmariaabb committed:
LPD-62010 buildService

carolmariaabb committed:
LPD-62010 Prevent users from subscribing to a folder or entry if they are already subscribed to its parent folder

carolmariaabb committed:
LPD-62010 If the user is already subscribed to a parent folder, do not display the subscribe and unsubscribe actions

mxs2 committed:
Revert "LPD-56406 Escape Parameter URL in RenderStructureFieldMVCResourceCommand"

This reverts commit 9e0026c8aa444937a2bfd079bcca35ab3dd18f5a.

mxs2 committed:
LPD-62034 Properly escape definition parameter for DDMFormField

algobob committed:
LPD-60753 Get scope as string

magjed4289 committed:
LPD-61755 Move test to reuse existing bundle processing logic

lfbesada committed:
LPD-62227 Prevent the redirect in case of not found error

andrea-ale-sbarra committed:
LPD-51865 sf

robertoDiaz committed:
LPD-61847 Check permissions

lfbesada committed:
LPD-62227 Add an integration tests to reproduce this use case

magjed4289 committed:
LPD-61755 Apply review suggestions

vendeltoreki committed:
LPD-61662 Add integration test

carolmariaabb committed:
LPD-62010 Update the logic for the subscription update action to exclude expired status

Without this condition, if you expire the entry, it will trigger two notifications

carolmariaabb committed:
LPD-62010 Return the object definition short name (without the `c_`)

carolmariaabb committed:
LPD-62010 Add integration tests

bcarvalho89 committed:
LPD-61847 Rename variable to hasAssignMembersPermission

bcarvalho89 committed:
LPD-61847 Remove hasAssignMembersPermission logic from ViewAllSpacesDisplayContext

carolmariaabb committed:
LPD-62010 Add new subscription notification template to notify when entries are added to folders

carolmariaabb committed:
LPD-62010 Add new subscription object action to notify when entries are added to folders

carolmariaabb committed:
LPD-62010 Notify all users subscribed to folders, subfolders, or entries

CaioERP committed:
LPD-58598 Sort JSP

algobob committed:
LPD-60753 Test

bcarvalho89 committed:
LPD-61847 Add a disabled input instead of hide the autocomplete input

carolmariaabb committed:
LPD-62010 Avoid NPE

JavierMoral committed:
LPD-61906 Add method that receives current group to be able to calculate the scopeExternalReferenceCode

Mikellorza committed:
LPD-62098 Add success messages for expire and restore

thiagobuarqque committed:
LPD-60753 consider the configuration category when searching

Mikellorza committed:
LPD-62098 buildLang

algobob committed:
LPD-60753 Use language service instead of LanguageUtil

rachaelkoestartyo committed:
LPD-56440 | LPD-62131 add endpoint /object-entry-histogram-metric

rachaelkoestartyo committed:
LPD-56440 | LPD-62131 buildREST

rachaelkoestartyo committed:
LPD-56440 | LPD-62131 implement

rachaelkoestartyo committed:
LPD-56440 | LPD-62131 add test

albertojml committed:
LPD-61120 Fix indentation

robertoDiaz committed:
LPD-61847 Handle actions in the FDS based in permissions

lluk4z committed:
LRSD-10662 Fix the article template

pedroliveirav committed:
LRSD-10217 Refactor LFU routes flow

bcarvalho89 committed:
LPD-61847 Check for assign permission based on item permission key

bcarvalho89 committed:
LPD-61847 Pass hasAssignMembersPermission to empty state

algobob committed:
LPD-60753 SF remove some blank lines

algobob committed:
LPD-60753 SF use try-with-resources for static mock

pedroliveirav committed:
LRSD-10217 Fix error component flow

liferay-continuous-integration commented:
To conserve resources, the PR Tester does not automatically run for every pull.

If your code changes were already tested in another pull, reference that pull in this pull so the test results can be analyzed.

If your pull was never tested, comment &quot;ci:test&quot; to run the PR Tester for this pull.

kenjiheigel commented:
ci:test:relevant

liferay-continuous-integration commented:
Test suite sf has been triggered on http://test-1-29

liferay-continuous-integration commented:
Test suite relevant has been triggered on http://test-1-40

liferay-continuous-integration commented:
<h1>Build started.</h1><p>Jenkins is currently running tests in the <code>relevant</code> test suite.</p><h4>Base Branch:</h4><p>Branch Name: <a href="https://github.com/liferay/liferay-portal/tree/master">master</a></p><h4>Job Summary:</h4><p>Job Link: <a href="https://test-1-40.liferay.com/job/test-portal-acceptance-pullrequest(master)/16013/">test-portal-acceptance-pullrequest(master)</a></p><p><b>For more details click <a href="https://test-1-40.liferay.com/userContent/jobs/test-portal-acceptance-pullrequest(master)/builds/16013/jenkins-report.html">here</a>.</b></p>

liferay-continuous-integration commented:
<div><div><strong>Jenkins Build:</strong><span></span><a href="https://test-1-29.liferay.com/job/test-portal-source-format/6849">test-portal-source-format#6849</a><br/></div><div><strong>Jenkins Report:</strong><span></span><a href="https://test-1-29.liferay.com/userContent/jobs/test-portal-source-format/builds/6849/jenkins-report.html">jenkins-report.html</a><br/></div><div><strong>Jenkins Suite:</strong><span></span>sf<br/></div><div><strong>Pull Request:</strong><span></span><a href="https://github.com/kenjiheigel/liferay-portal/pull/1921">kenjiheigel#1921</a><br/></div><div><strong>Testray Routine:</strong><span></span><a href="https://testray.liferay.com/#/project/35392/routines/45357">EE Pull Request</a><br/></div><div><strong>Testray Build:</strong><span></span><a href="https://testray.liferay.com/#/project/35392/routines/45357/build/275801102">[master] ci:test:sf - kenjiheigel &gt; kenjiheigel - PR#1921 - 2025-08-01[12:30:59]</a><br/></div><div><strong>Testray Build ID:</strong><span></span><code>275801102</code><br/></div><div><strong>Testray Importer:</strong><span></span><a href="https://test-1-29.liferay.com/job/test-portal-source-format/6849/">test-portal-source-format#6849</a><br/></div></div>

liferay-continuous-integration commented:
<div><div><strong>Jenkins Build:</strong><span></span><a href="https://test-1-40.liferay.com/job/test-portal-acceptance-pullrequest(master)/16013">test-portal-acceptance-pullrequest(master)#16013</a><br/></div><div><strong>Jenkins Report:</strong><span></span><a href="https://test-1-40.liferay.com/userContent/jobs/test-portal-acceptance-pullrequest(master)/builds/16013/jenkins-report.html">jenkins-report.html</a><br/></div><div><strong>Jenkins Suite:</strong><span></span>relevant<br/></div><div><strong>Pull Request:</strong><span></span><a href="https://github.com/kenjiheigel/liferay-portal/pull/1921">kenjiheigel#1921</a><br/></div><div><strong>Testray Routine:</strong><span></span><a href="https://testray.liferay.com/#/project/35392/routines/45357">EE Pull Request</a><br/></div><div><strong>Testray Build:</strong><span></span><a href="https://testray.liferay.com/#/project/35392/routines/45357/build/275902813">[master] ci:test:relevant - kenjiheigel &gt; kenjiheigel - PR#1921 - 2025-08-01[12:31:03]</a><br/></div><div><strong>Testray Build ID:</strong><span></span><code>275902813</code><br/></div><div><strong>Testray Importer:</strong><span></span><a href="https://test-1-40.liferay.com/job/test-portal-acceptance-pullrequest(master)/16013/">test-portal-acceptance-pullrequest(master)#16013</a><br/></div></div>

interaminense committed:
LPD-60567 Follow Up, rename testids to fix playwright tests

dsitu committed:
LPD-62013 Fix test

BryanEngler committed:
LPD-50745 portal-search-elasticsearch7-impl: Move translator to legacy package and refactor test

https://liferay.atlassian.net/browse/LPD-50745

BryanEngler committed:
LPD-50745 portal-search-opensearch2-impl: Explicitly set sort order and refactor test

https://liferay.atlassian.net/browse/LPD-50745

BryanEngler committed:
LPD-50745 portal-search-solr8-impl: Refactor test

https://liferay.atlassian.net/browse/LPD-50745

brianchandotcom committed:
LPD-61120 SF

brianchandotcom committed:
LPD-61120 I just added line breaks so I could read this better. This if check looks wrong?

gabrielwas committed:
LPD-62223 Fix playwright tests related to portuguese labels

albertojml committed:
LPD-61120 Fix condition

lfbesada committed:
LPD-62227 Ensure the same behavior no matter the configuration or the user role

ccorreagg committed:
LPD-62249 No plural word ends with 'tus' in english

brianchandotcom committed:
LPD-61120 prep next

brianchandotcom committed:
LPD-61120 prep next

ccorreagg committed:
LPD-62249 Add /status endpoint to test it

ccorreagg committed:
LPD-62249 buildREST (portal-tools-rest-builder-test-impl module)

jkappler committed:
LPD-62253 We cannot guarantee the values of the configuration for the test. And the changing of the configuration is covered in testIsAutogenerateDDMStructureKeyConfiguredso it’s better to remove it

lfbesada committed:
LPD-61879 Rename

lfbesada committed:
LPD-61879 SF Auto

lfbesada committed:
LPD-61879 Use the draft layout

lfbesada committed:
LPD-61879 Move to a field

vendeltoreki committed:
LPD-61662 Separate _publishLayouts() and _publishAllLayouts() methods

lfbesada committed:
LPD-61879 Simplify

lfbesada committed:
LPD-61879 Simplify naming

lfbesada committed:
LPD-61879 Improve assertions

lfbesada committed:
LPD-61879 Early return in case of not null portlet ID since it won't have other editable values

ccorreagg committed:
LPD-62249 Apply this if compatibilityVersion >= 11 to avoid possible breaking changes

ccorreagg committed:
LPD-62249 Update compatibilityVersion

# breaking

## What modules/util/portal-tools-rest-builder/src/main/java/com/liferay/portal/tools/rest/builder/internal/freemarker/util/OpenAPIUtil.java

Compatibility version has been bumped up with the introduction of the fix of the operationId for certain words ("Status").

## Why

REST builder autogenerate the operationId based on the path, we need to update it so it might cause breaking changes

----

liferay-translation committed:
LPS-77699 Update Translations

lfbesada committed:
LPD-59858 Restore the testing of assigning no categories, otherwise we aren't testing the case of unassigning all the categories

magjed4289 committed:
LPD-61755 Avoid chaining and sending an empty array instead of an OD (we only really need to check the task configuration data), but can't remove try-finally, cause if this test fails then the user by default will stay deactivated and it can affect other tests

lfbesada committed:
LPD-59858 Instead of create vocabularies in the global site, override the base randomIrrelevantSitePage method

lfbesada committed:
LPD-59858 Adapt usages

brianchandotcom committed:
LPD-61755 Avoid finally

brianchandotcom committed:
LPD-61755 Rename

brianchandotcom committed:
LPD-61755 Simplify

brianchandotcom committed:
LPD-61755 Simplify

brianchandotcom committed:
LPD-61755 auto SF

brianchandotcom committed:
LPD-61185 SF

brianchandotcom committed:
LPD-61185 Order here doesn't matter

bcarvalho89 committed:
LPD-61847 Fix duplicate view members item

interaminense committed:
LPD-60567 Follow Up, remove unused lang and add missing ones

interaminense committed:
LPD-60567 buildLang

pedroliveirav committed:
LRSD-10217 Add translations

luisacmn committed:
LRSD-10672 Define Announcement Image Type object definition

interaminense committed:
LPD-60567 Fix language to be a complete sentence

interaminense committed:
LPD-60567 buildLang

brianchandotcom committed:
LPD-61662 SF

joseabelenda committed:
LRSD-10672 SF

brianchandotcom committed:
LPD-58598 Consistent SF

brianchandotcom committed:
LPD-59858 SF

ccorreagg committed:
LPD-62249 Ignore test, this is not the purpose of the test, just to check the operationId generated

ccorreagg committed:
LPD-62249 Sort

Co-authored-by: Daniel Raposo <daniel.raposo@liferay.com>

jaredgorski committed:
LPD-61840 Implement AWS UX terraform improvements

bcarvalho89 committed:
LPD-61847 Invert permission logic return

marcoscv-work committed:
LPD-62271 - Remove Horizontal Scrollbar

marcoscv-work committed:
LPD-62270 - Add New Space Responsive Version

pyoo47 committed:
LRCI-6178 StringBuilder.isEmpty() doesn't exist prior to JDK 15

carolmariaabb committed:
LPD-62304 Add new integration test to expose issue

carolmariaabb committed:
LPD-62304 Handle cases where a datetime value has 22 characters but doesn't follow the ISO 8601 pattern

joseabelenda committed:
LRSD-10662 SF

ChrisKian committed:
LPD-56434 Partially revert the commit 'LPD-56434 Simplify 28338b5ed', otherwise Group and User ResourceTest classes will fail

pyoo47 committed:
LPD-60861 Upgrade to version 1.12.0

brianchandotcom committed:
LPD-60861 prep next

brianchandotcom committed:
LPD-61840 SF

brianchandotcom committed:
LPD-62227 Set only when needed

brianchandotcom committed:
LPD-60567 Wordsmith

brianchandotcom committed:
LPD-60567 Regen

brianchandotcom committed:
LPD-60753 SF

ling-alan-huang committed:
LPD-61120 Auto SF

ling-alan-huang committed:
LPD-62227 Buy space

ling-alan-huang committed:
LPD-62227 Buy space

ling-alan-huang committed:
LPD-62227 SF

brianchandotcom committed:
LPD-62010 Assume the test works

brianchandotcom committed:
LPD-62010 Simplify

brianchandotcom committed:
LPD-61879 As used

brianchandotcom committed:
LPD-61879 auto SF

brianchandotcom committed:
LPD-62249 prep next

brianchandotcom committed:
LPD-62249 prep next

ambrinchaudhary committed:
LPD-62332 Adds new property fixing the tests

jkappler committed:
LPD-62335 Unlocalized element has only one locale, so update test

liferay-translation committed:
LPS-77699 Update Translations

Mikellorza committed:
LPD-62098 Add delete entry version confirmation modal

lfbesada committed:
LPD-61906 Implement the default method

lfbesada committed:
LPD-61906 Add a new method to get the info item but receiving the current scope group ID as a parameter

lfbesada committed:
LPD-61906 Baseline

jkappler committed:
LPD-62332 Instead of DepotEntry, listen for Group, since in tests one can be deleted before the other

victorg1991 committed:
LPD-62371 baseline

BryanEngler committed:
LPD-50745 Update param names

https://liferay.atlassian.net/browse/LPD-50745

rafaprax committed:
LPD-56434 SF

ryanschuhler committed:
LRSD-10044 Fix cp button link mapping

ryanschuhler committed:
LRSD-10044 Add conditional container fragments

abfranca committed:
LRSD-10158 Move sort logic to frontend

ryanschuhler committed:
LRSD-9453 Add alternate article templates

abfranca committed:
LRSD-10158 Add default icon for menu item

ryanschuhler committed:
LRSD-10122 Add search blueprints

ryanschuhler committed:
LRSD-10122 Fix deselect function

ryanschuhler committed:
LRSD-10122 Fix item sort

brianchandotcom committed:
LPD-60753 Fix test while keeping the SF

brianchandotcom committed:
LRSD-10430 Same thing?

brianchandotcom committed:
LRCI-6178 prep next

brianchandotcom committed:
LPD-62098 Wordsmith

brianchandotcom committed:
LPD-62098 Regen

brianchandotcom committed:
LPD-62332 Can this ever be null?

brianchandotcom committed:
LPD-50745 Sort

kenjiheigel committed:
TEMP try this

kenjiheigel commented:
cloud:ci:test:relevant
